phpseclib/File/X509.php

```diff
@@ -3794,7 +3794,7 @@ function &_subArray(&$root, $path, $create = false)
      * @param string $path optional absolute path with / as component separator
      * @param bool $create optional
      * @access private
-     * @return array ref or false
```

Comment:
Hmmmm. `&array` is invalid syntax as far as I'm aware. At least, I've never seen it used before, and I know of phpdoc parsers that would fail to read that correctly.

> Why mixed instead of array|false like you have below?

My bad there. I'll change that now.